packages/plugin-utils/src/regexes.ts

```diff
@@ -1,5 +1,5 @@
 export const regexQuotedString = /(["'`])((?:\\\1|(?:(?!\1)|\n).)*?)\1/mg
-export const regexClassCheck = /^[a-z.-]+[\w:/\\.$-]*\w$/
+export const regexClassCheck = /^[\w.+-]+[\w:/\\.$-]*\w$/
```

Comment:
Does it have to be `\w`? I think we can assume there is not leading uppercase letters and numbers.

Reply:
Hum, nvm.